Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/EditableRowSupport/EditableRow.cs

```diff
@@ -302,13 +311,15 @@ protected void PrepareValidation (int columnIndex)
     {
       if (HasEditControl (columnIndex))
       {
-        IBusinessObjectBoundEditableWebControl editControl = GetEditControl (columnIndex);
+        IBusinessObjectBoundEditableWebControl editControl = GetEditControl (columnIndex)!;
```

Comment:
Assertion

Reply:
In Data, we have the pattern of appending "Mandatory" for a not-null variant of an api. Maybe it's a good idea (follow-up task) to add private methods for GetEditControl and GetValidators that return not-null values and check with an exception if it is null. That way, we can get rid of the need for inline-assertions.

-> GetEditControlMandatory(...)
-> GetValidatorsMandatory(...)